### PR TITLE
feat(php82): extensions support for php 8.2

### PR DESCRIPTION
**Pull request by shouze**
and upgrade bref base image

### CODE REVIEW

**Comment by Nyholm** [comment]
Thank you. 
It seams like the xdebug layer is failing. Do you know why?

**Comment by shouze** [comment]
@Nyholm yes sorry, was probably related to https://github.com/xdebug/xdebug/pull/851. I try to update the PR this week as php 8.2 is coming soon, first RCs.

**Comment by shouze** [comment]
@Nyholm should be alright now.

**Comment by shouze** [comment]
@Nyholm oh well...
```
#6 3.244 checking Check for supported PHP versions... configure: error: not supported. Need a PHP version >= 8.0.0 and < 8.3.0 (found 7.3.33)
```

Should we be BC break & remove support of php 7.3? So this should be version 0.12.0 or better 1.0.0 no?

File: layers/xdebug/config.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
     "php": [
-        "73",
```
Comment (shouze) [inline layers/xdebug/config.json:3]: @Nyholm I've dropped support of php 7.3

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+- Using bref/build-php-XX:1.7.4 as base image
+
+### Added
+
+- Extensions support for PHP 8.2, all except following ones:
+  - blackfire
+  - cassandra
+  - memcache
+  - newrelic
+  - newrelic
```
Comment (shouze) [inline CHANGELOG.md:16]:
```suggestion
```

**Comment by shouze** [comment]
@Nyholm LGTM

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,24 @@
...
 
+## 0.11.35
+
+### Changed
+
+- Using bref/build-php-XX:1.7.4 as base image
+
+### Added
+
+- Extensions support for PHP 8.2, all except following ones:
```
Comment (Nyholm) [inline CHANGELOG.md:11]: Could we list the ones we changed instead?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,24 @@
 # Change log
 
+## 0.11.35
```
Comment (Nyholm) [inline CHANGELOG.md:3]: Let's do a new minor

File: layers/xdebug/config.json (hunk 1 above)
Comment (Nyholm) [reply layers/xdebug/config.json:3]: 👍🏽

**Review by Nyholm — COMMENTED** [review]
Thanky ou

**Comment by shouze** [comment]
@Nyholm sorry for the delay, should be all good right now. Also, as php 7 is fully deprecated from now, I can make another PR to completely drop php 7 support after this one. WDYT?

**Comment by Nyholm** [comment]
Let's merge this and drop PHP7 in next PR.

**Comment by Nyholm** [comment]
Just released `0.11.35` which will be the last 0.11.* release

**Review by Nyholm — APPROVED** [review]

**Comment by Nyholm** [comment]
I tried to rebase this PR but I wasn't allowed to push to your branch. Could you please rebase so I can merge?

**Comment by shouze** [comment]
> I tried to rebase this PR but I wasn't allowed to push to your branch. Could you please rebase so I can merge?

Done, and bumped to 0.12.0 in Changelog

**Pull request merged**